### ISSUE TITLE
test_pricer.py giving wrong answer

### ISSUE THREAD

**Issue by sandropa** [scipopt/PySCIPOpt#851]
**Describe the bug**
Function `cuttingstock()` is not giving the correct answer for the following example:
```
# item widths
widths = [3, 2, 1]
# width demand
demand = [3, 4, 3]
# roll length
rollLength = 4
```

The number of rolls needed is 6. However, the function gives the answer 5.

**To Reproduce**
Comment the lines:
```
# item widths
widths = [14, 31, 36, 45]
# width demand
demand = [211, 395, 610, 97]
# roll length
rollLength = 100
```

Replace them with:
```
# item widths
widths = [3, 2, 1]
# width demand
demand = [3, 4, 3]
# roll length
rollLength = 4
```
Also, comment the lines that start with `assert`.

**Expected behavior**
The sum of used cutting patterns should be 6.

**Comment by Joao-Dionisio** [comment]
Hello, @sandropa! Why do you say that it's wrong? The given solution seems to work:

![image](https://github.com/scipopt/PySCIPOpt/assets/57299939/ca863e3a-df83-4792-9617-0077e6d34555)

Having 

[item2, item2] + [item2, item2] + [item1, item3] + [item1, item3] + [item1, item3] 

seems like it does the trick. 4 item2s, 3 item1s, 3 item3s. The length of each roll is 4 and the total number of rolls is 5.

**Comment by sandropa** [comment]
The answers is wrong for the demand [3, 4, 4].
![image](https://github.com/scipopt/PySCIPOpt/assets/134156003/747dbca3-702e-4bd7-8006-44fb68126555)

**Comment by Joao-Dionisio** [comment]
Thank you for this issue @sandropa! There was a mistake in displaying the solution. The actual solution is this:

![image](https://github.com/scipopt/PySCIPOpt/assets/57299939/6c820a68-085a-4ed6-8970-eaa104697f54)

(We are using 0.25 of a roll because this is only the linear relaxation)

The pricer itself seems correct, but there a couple of things that warrant changes.

**Comment by Joao-Dionisio** [comment]
Displayed solution fixed in #852.

I think that's everything, @sandropa. I'll be closing the issue, but if you feel like there are other things wrong, don't be afraid to reopen!
